packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -29,6 +28,9 @@ import {
   PieChartSvgProps,
   PieChartType,
 } from './types';
+import { getTooltipContainerStyles } from '../../themes/mixins';
+import { SeriesTable, SeriesTableRowProps } from '../Graph/GraphTooltip/SeriesTable';
+import { GraphTooltipOptions } from '../Graph/GraphTooltip/models.gen';
```

Comment:
I think we can rename the GraphTooltipOptions to VizTooltipOptions (similar to VizLegendOptions).

Next up would be to move Tooltips related things (Chart/Tooltip) to VizTooltip directory, rename it to VizTooltip and NOT export it under Chart namespace. This way we start unifying the convention about viz addons. I think the one introduced by @torkelo with VizLegend is a good way forward. 

WDYT?